From: Samuel Henrique <samueloph@debian.org>
Date: Mon, 17 Aug 2026 22:21:27 -0700
Subject: tests: fix Content-Length mismatch in test2064

Spotted when packaging rc2 for Debian, other similar tests (2061-2066)
already have `crlf="headers"` for the Server-side.

Forwarded: yes, https://github.com/curl/curl/pull/22611
Last-Update: 2026-08-17
---
 tests/data/test2064 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/data/test2064 b/tests/data/test2064
index d389753..0ab4048 100644
--- a/tests/data/test2064
+++ b/tests/data/test2064
@@ -9,7 +9,7 @@ HTTP Digest auth
 </info>
 # Server-side
 <reply>
-<data crlf="yes">
+<data crlf="headers">
 HTTP/1.1 401 Authorization Required
 Server: Apache/1.3.27 (Darwin) PHP/4.1.2
 WWW-Authenticate: Digest realm="testrealm", nonce="2053604145", algorithm="SHA-256"
